tests/bin/test_tmux_layout_integration.py

```
@@ -1,289 +1,289 @@
+            # 初期ペイン数を記録
+            cmd = f"tmux list-panes -t {test_session}:MAIN -F '#{{pane_index}}'"
+            result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
+            initial_count = len(result.stdout.strip().split('\n'))
+            print(f"  Initial pane count: {initial_count}")
+            
+            # 5つの子エージェントを順次追加
+            for i in range(1, 6):
+                task_id = f"T{i:03d}"
+                (tmppath / "work" / task_id).mkdir(parents=True)
+                
+                pane = busd.spawn_child(task_id, f"work/{task_id}",
```

Ruff (0.12.2)

Local variable `pane` is assigned to but never used

Remove assignment to unused variable `pane`

(F841)